gcloud-java-storage/src/main/java/com/google/gcloud/storage/Storage.java

```diff
@@ -755,12 +829,26 @@ public static Builder builder() {
   BlobInfo compose(ComposeRequest composeRequest);
 
   /**
-   * Send a copy request.
+   * Sends a copy request. Returns a {@link CopyWriter} object for the provided
+   * {@code CopyRequest}. If source and destination objects share the same location and storage
+   * class the source blob is copied and its information can be accessed with
+   * {@link CopyWriter#result()}, regardless of the {@link CopyRequest#megabytesRewrittenPerCall}
+   * parameter. If source and destination have different location or storage class multiple RPC
+   * calls might be needed depending on blob's size.
+   * <p>
+   * Example usage of copy:
+   * <pre>    {@code CopyWriter copyWriter = service.copy(copyRequest);}
+   *    {@code while(!copyWriter.isDone()) {
+   *       copyWriter.copyChunk();
+   *   }}
```

Comment:
Do you think we should include the BlobInfo blobInfo = copyWriter.result() in the code sample?

Also, why do you use 2 different {@code } tags?

If you think that result should block (or that we should provide a more explicit method for it) than maybe it is worth showing it as well...